pkg/datarepair/repairer/config.go

```diff
@@ -28,6 +28,15 @@ type Config struct {
 	Interval     time.Duration `help:"how frequently checker should audit segments" default:"3600s"`
 	miniogw.ClientConfig
 	miniogw.RSConfig
+	nodeStats nodeStats
+}
+
+type nodeStats struct {
```

Comment:
It doesn't look like you use `Excluded` here either. `Uptime`/`AuditSuccess` -> `float64`. You should also add help messages similar to the ones you have in `pkg/miniogw/config.go`